plugins/org.jboss.reddeer.eclipse/src/org/jboss/reddeer/eclipse/condition/ServerExists.java

```diff
@@ -58,4 +62,13 @@ public String description() {
 		return "there is server with name: " + this.name;
 	}
 	
+	/* (non-Javadoc)
+	 * @see org.jboss.reddeer.common.condition.WaitCondition#errorMessage()
+	 */
+	@Override
+	public String errorMessage() {
```

Comment:
we should have a different message for Until and While

While error - Server still exists
Until - Server has not been found (the current error message)